<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUser</class>
 <widget class="QMainWindow" name="NewUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>610</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="registerBtn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>532</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1280</width>
      <height>731</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Login-Register.qrc">:/Pages/All_Materials/Register_pg.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>315</y>
      <width>195</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(229, 223, 206, 0.5);
color: #000000; 
border: 2px solid rgba(229, 223, 206, 0.5); 
font-weight: 600; 

</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNameLineEdit">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>269</y>
      <width>257</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(229, 223, 206, 0.5);
color: #000000; 
border: 2px solid rgba(229, 223, 206, 0.5); 
font-weight: 600; 

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>175</y>
      <width>257</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(229, 223, 206, 0.5);
color: #000000; 
border: 2px solid rgba(229, 223, 206, 0.5); 
font-weight: 600; 

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idLineEdit">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>361</y>
      <width>257</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(229, 223, 206, 0.5); 
color: #000000; 
border: 2px solid rgba(229, 223, 206, 0.5); 
font-weight: 600; 

</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sectionSpinBox">
    <property name="geometry">
     <rect>
      <x>705</x>
      <y>457</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(229, 223, 206, 1);
color: #000000; 
border: 2px solid rgba(229, 223, 206, 0.5); 
font-weight: 600; 

</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="female">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>497</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #CE2AB6; 


</string>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
    <property name="icon">
     <iconset resource="Login-Register.qrc">
      <normaloff>:/Icons/All_Materials/Icons/Female Icon.png</normaloff>:/Icons/All_Materials/Icons/Female Icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="male">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>497</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #CE2AB6; 


</string>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
    <property name="icon">
     <iconset resource="Login-Register.qrc">
      <normaloff>:/Icons/All_Materials/Icons/Male Icon.png</normaloff>:/Icons/All_Materials/Icons/Male Icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="programComboBox">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>408</y>
      <width>257</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(229, 223, 206, 1);
color: #000000; 
border: 2px solid rgba(229, 223, 206, 0.5); 
font-weight: 600; 

</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="yearSpinBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>457</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(229, 223, 206, 1);
color: #000000; 
border: 2px solid rgba(229, 223, 206, 0.5); 
font-weight: 600; 

</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailLineEdit">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>222</y>
      <width>257</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(229, 223, 206, 0.5);
color: #000000; 
border: 2px solid rgba(229, 223, 206, 0.5); 
font-weight: 600; 

</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showPassCheckBox">
    <property name="geometry">
     <rect>
      <x>744</x>
      <y>321</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>loginBtn</zorder>
   <zorder>registerBtn</zorder>
   <zorder>passwordLineEdit</zorder>
   <zorder>userNameLineEdit</zorder>
   <zorder>nameLineEdit</zorder>
   <zorder>idLineEdit</zorder>
   <zorder>sectionSpinBox</zorder>
   <zorder>female</zorder>
   <zorder>male</zorder>
   <zorder>programComboBox</zorder>
   <zorder>yearSpinBox</zorder>
   <zorder>emailLineEdit</zorder>
   <zorder>showPassCheckBox</zorder>
  </widget>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>userNameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>idLineEdit</tabstop>
  <tabstop>programComboBox</tabstop>
  <tabstop>yearSpinBox</tabstop>
  <tabstop>sectionSpinBox</tabstop>
  <tabstop>male</tabstop>
  <tabstop>female</tabstop>
  <tabstop>registerBtn</tabstop>
  <tabstop>loginBtn</tabstop>
  <tabstop>showPassCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="Login-Register.qrc"/>
 </resources>
 <connections/>
</ui>
